src/common/SearchBar.js

```diff
@@ -0,0 +1,66 @@
+import Head from "next/head";
+import Image from "next/image";
+import { Inter } from "next/font/google";
+// import styles from "@/styles/Home.module.css";
+import FooterBar from "@/common/FooterBar";
+import ReusableNavLink from "@/common/ReusableNavLink";
+import Link from 'next/link';
+import Data from "../mock-data.json"
+import styles from "@/styles/SearchBar.module.css";
+import {useState} from 'react';
+
+
+function SearchBar({ placeholder, data }) {
+
+    const [filteredData, setFilteredData] = useState([]);
+
+    const handleFilter = (event) => {
+        const searchWord = event.target.value
+        const newFilter = data.filter((value) => {
+            return value.name.toLowerCase().includes(searchWord.toLowerCase());
+        });
+        if (searchWord === "") {
+            setFilteredData([])
+        } else {
+            setFilteredData(newFilter)
+        } 
+    };
+    return (
+        <div>
+            <form>
+                <div>
+                    <div className="search-inputs">
+                    <input id="search-input-bar" type="text" placeholder= {placeholder} name='search' required onChange={handleFilter} />
+                    <button type='submit'>
+                    <Image
+                        src="/search-glass.svg"
+                        alt="Search Image"
+                        className={styles.searchImage}
+                        width={20}
+                        height={20}
+                    />
+                    </button>
+                    </div>
+                </div>
+            </form>
+            {filteredData.length != 0 && (
+                <div className={styles.dataResults}>
```

Comment:
Make this absolute with a z-index >10 and also with a top of about -64px (Just enough pixel to appear underneath the input box

Reply:
When using the above css styling, the input box was no longer visible, changed some styling settings to appear below the input box.